docs/add-ins/addin-tutorial.md

```diff
@@ -364,7 +457,7 @@ That takes care of the UI for the dialog, but now we need to add code to make it
         }
       });
 
-      // When the Done button is clicked, send the
+      // When the Done button is chosen, send the
```

Comment:
instead of "chosen", perhaps "pressed"

Reply:
Good suggestion; will make this update throughout.